src/util/heap/gc_trigger.rs

```diff
@@ -79,6 +79,11 @@ impl<VM: VMBinding> GCTrigger<VM> {
 /// GC start/end so they can collect some statistics about GC and allocation. The policy needs to
 /// decide the (current) heap limit and decide whether a GC should be performed.
 pub trait GCTriggerPolicy<VM: VMBinding>: Sync + Send {
+    /// Inform the triggering policy that we have pending allocation.
+    /// Any policy with dynamic heap size should take this into account when calculating a new heap size.
```

Comment:
We use the word "policy" for both spaces (CopySpace, ImmixSpace, etc.) and GC trigger (GCTriggerPolicy).  But because this method is called by Space, I thought the "policy" here refers to spaces when I first read this sentence.

```suggestion
    /// Any GC trigger policy with dynamic heap size should take this into account when calculating a new heap size.
```